roles/node/tasks/add-validator.yml

```diff
@@ -0,0 +1,76 @@
+---
+- name: Get P chain address
+  uri:
+    url: "http://{{ avalanchego_http_host }}:{{ avalanchego_http_port }}/ext/bc/P"
+    method: POST
+    body_format: json
+    body:
+      jsonrpc: "2.0"
+      method: platform.listAddresses
+      params:
+        username: "{{ avalanche_node_username }}"
+        password: "{{ avalanche_node_password }}"
+      id: 1
+  register: list_addresses_res
+  failed_when: list_addresses_res.json.error is defined or list_addresses_res.json.result.addresses | length < 1
+
+- name: Get current P chain height
+  uri:
+    url: "http://{{ avalanchego_http_host }}:{{ avalanchego_http_port }}/ext/bc/P"
+    method: POST
+    body_format: json
+    body:
+      jsonrpc: "2.0"
+      method: platform.getHeight
+      id: 1
+  register: get_height_res
+  failed_when: get_height_res.json.error is defined
+
+- name: Get current P chain validators
+  uri:
+    url: "http://{{ avalanchego_http_host }}:{{ avalanchego_http_port }}/ext/bc/P"
+    method: POST
+    body_format: json
+    body:
+      jsonrpc: "2.0"
+      method: platform.getValidatorsAt
+      params:
+        height: "{{ get_height_res.json.result.height }}"
+        subnetID: "11111111111111111111111111111111LpoYY"
```

Comment:
I think we can omit the `subnetID` as it defaults to the Primary network.

Reply:
Also maybe using [getCurrentValidators](https://docs.avax.network/apis/avalanchego/apis/p-chain#platformgetcurrentvalidators) is easier and we can just query the `nodeID` we are interested in.

Reply:
Good catch, I'll remove the getHeight and replace getValidatorsAt by getCurrentValidators

Reply:
Done in the last commit